pkg/disk/cloud.go

```diff
@@ -234,7 +234,7 @@ func (ad *DiskAttachDetach) attachDisk(ctx context.Context, diskID, nodeID strin
 
 	// step 5: diff device with previous files under /dev
 	if fromNode {
-		device, err := DefaultDeviceManager.GetDeviceByVolumeID(diskID)
+		device, err := DefaultDeviceManager.WaitDevice(ctx, diskID)
```

Comment:
Wait 10 seconds for each disk without a series id?

Reply:
Better to add logs to record err message

Reply:
> Wait 10 seconds for each disk without a series id?

Currently yes. This can be resolved by #1199 , which explicitly distinguishes disks with or without serial number.

Reply:
Added logs for all abnormal code path.